Index: plymouth-0.8.6.1/systemd-units/systemd-ask-password-plymouth.path.in
===================================================================
--- plymouth-0.8.6.1.orig/systemd-units/systemd-ask-password-plymouth.path.in
+++ plymouth-0.8.6.1/systemd-units/systemd-ask-password-plymouth.path.in
@@ -1,6 +1,5 @@
 [Unit]
 Description=Forward Password Requests to Plymouth Directory Watch
-Documentation=http://www.freedesktop.org/wiki/Software/systemd/PasswordAgents
 DefaultDependencies=no
 Conflicts=shutdown.target
 After=plymouth-start.service
Index: plymouth-0.8.6.1/systemd-units/systemd-ask-password-plymouth.service.in
===================================================================
--- plymouth-0.8.6.1.orig/systemd-units/systemd-ask-password-plymouth.service.in
+++ plymouth-0.8.6.1/systemd-units/systemd-ask-password-plymouth.service.in
@@ -1,6 +1,5 @@
 [Unit]
 Description=Forward Password Requests to Plymouth
-Documentation=http://www.freedesktop.org/wiki/Software/systemd/PasswordAgents
 DefaultDependencies=no
 Conflicts=shutdown.target
 After=plymouth-start.service
